tbx/project_styleguide/templates/patterns/molecules/listing/listing--image.html

```diff
@@ -0,0 +1,42 @@
+{# Designed to be included from another template and values passed in #}
+{% load wagtailimages_tags %}
+
+{% image listing_image fill-370x370 format-webp as desktop_image %}
+{% image listing_image fill-740x740 format-webp as desktop_image_retina %}
+{% image listing_image fill-370x335 format-webp as mobile_image %}
+{% image listing_image fill-740x670 format-webp as mobile_image_retina %}
+
+<li class="listing-image listing">
+    <div>
+        {# alt text is empty for consistency with listing--avatar.html where the image is inside the link, and therefore not announced #}
+        <picture>
+            <source media="(max-width: 598px)" srcset="{{  mobile_image.url }} 1x, {{ mobile_image_retina.url }} 2x" />
+            <source media="(min-width: 599px)" srcset="{{ desktop_image.url }} 1x, {{ desktop_image_retina.url }} 2x" />
+            <img src="{{ desktop_image.url }}" alt="" loading="lazy" class="listing-image__image" />
+        </picture>
+    </div>
+
+    <div>
+        <a class="listing-image__link listing__link" href="{{ link }}" aria-label="{{ aria_text|default:title }}">
```

Comment:
I'm not seeing the focus ring for this component but oddly I can see it for the listing--avatar. I'll have a look to see if I can spot the issue!

Reply:
Making the anchor display: block/inline-block seems to resolve it - I wonder if it needs it because it's part of a flex layout?

Reply:
Updated in 777f9426ac4fad26fabbf1d04011880618985534